sw/device/tests/keymgr_sideload_aes_test.c

```diff
@@ -20,25 +20,40 @@
 
 #define TIMEOUT (1000 * 1000)
 
+// Plaintext 16B block matching that input to the AES C model =
+// 00112233445566778899aabbccddeeff Refer to hw/ip/aes/rtl/aes_cipher_core.sv
```

Comment:
```suggestion
// `00112233445566778899aabbccddeeff` Refer to `hw/ip/aes/rtl/aes_cipher_core.sv`
```
Reference [here](https://docs.opentitan.org/doc/sg/c_cpp_coding_style/#comments)